staging/src/k8s.io/client-go/tools/cache/store.go

```diff
@@ -199,17 +199,17 @@ func (c *cache) Index(indexName string, obj interface{}) ([]interface{}, error)
 	return c.cacheStorage.Index(indexName, obj)
 }
 
-func (c *cache) IndexKeys(indexName, indexKey string) ([]string, error) {
-	return c.cacheStorage.IndexKeys(indexName, indexKey)
+func (c *cache) IndexKeys(indexName, indexedValue string) ([]string, error) {
+	return c.cacheStorage.IndexKeys(indexName, indexedValue)
 }
 
 // ListIndexFuncValues returns the list of generated values of an Index func
 func (c *cache) ListIndexFuncValues(indexName string) []string {
 	return c.cacheStorage.ListIndexFuncValues(indexName)
 }
 
-func (c *cache) ByIndex(indexName, indexKey string) ([]interface{}, error) {
```

Comment:
add `ByIndex returns the stored objects whose set of indexed values for the named index includes the given indexed value.` godoc here, this signature is really hard to understand otherwise